EasyPost.Tests/PickupTest.cs

```diff
@@ -1,110 +1,117 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.Threading.Tasks;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
+using EasyPost.Models.API;
+using EasyPost.Utilities.Annotations;
+using Xunit;
 
 namespace EasyPost.Tests
 {
-    [TestClass]
-    public class PickupTest
+    public class PickupTest : UnitTest
     {
-        private TestUtils.VCR _vcr;
-
-        [TestInitialize]
-        public void Initialize()
+        public PickupTest() : base("pickup")
         {
-            _vcr = new TestUtils.VCR("pickup");
         }
 
-        private static async Task<Pickup> CreateBasicPickup()
-        {
-            Shipment shipment = await Shipment.Create(Fixture.OneCallBuyShipment);
-            Dictionary<string, object> pickupData = Fixture.BasicPickup;
-            pickupData["shipment"] = shipment;
-            return await Pickup.Create(pickupData);
-        }
+        #region CRUD Operations
 
-        [TestMethod]
+        [Fact]
+        [CrudOperations.Create]
         public async Task TestCreate()
         {
-            _vcr.SetUpTest("create");
+            UseVCR("create");
 
             Pickup pickup = await CreateBasicPickup();
 
-            Assert.IsInstanceOfType(pickup, typeof(Pickup));
-            Assert.IsTrue(pickup.id.StartsWith("pickup_"));
-            Assert.IsNotNull(pickup.pickup_rates);
+            Assert.IsType<Pickup>(pickup);
+            Assert.StartsWith("pickup_", pickup.id);
+            Assert.NotNull(pickup.pickup_rates);
         }
 
-        [TestMethod]
-        public async Task TestRetrieve()
+        [Fact]
+        [CrudOperations.Read] // not really?
```

Comment:
Question/Issue: What is the not really comment for? Comments that don't provide any context shouldn't be kept around. Can we either clarify the comment with something meaningful or remove it completely?